docs/getting_started.md

```diff
@@ -30,35 +32,31 @@ If you see any changes between the two feel free to create a
 [pull request](https://github.com/google/flax/compare)
 to synchronize this Colab with the actual example.
 
++++ {"id": "nwJWKIhdwxDo"}
+
+## 1. Install Flax
```

Comment:
Should we remove the numbering (1, 2, ...)? Doesn't add much to the document IMO but makes it annoying to maintain.

Reply:
IMO, it helps follow all the steps especially for new Flax users.